stdlib/compile.go

```diff
@@ -0,0 +1,125 @@
+// +build ignore
+
+package main
```

Comment:
What is this cmd for? Is it just a sanity check that it works? Should we move it to the `cmd` package?

Reply:
It's used by `go generate` to create the flatbuffers files. Maybe I should try to integrate this behavior into the `builtin` command instead?